src/__stories__/Index.stories.mdx

```diff
@@ -1,7 +1,46 @@
-import { Description, Meta } from '@storybook/addon-docs/blocks'
+import { Meta } from '@storybook/addon-docs/blocks'
 
 <Meta title="Home" />
 
 # Welcome to Scaleway UI
 
-<Description />
+Scaleway UI is a set of React components you can use to build your application. It is used to build our [Scaleway Console](https://console.scaleway.com).
+
+## Getting started
+
+### Installation
+
+With npm:
+
+```bash
+npm install @scaleway/ui
+```
+
+With yarn:
+
+```bash
+yarn add @scaleway/ui
+```
+
+### Add a ThemeProvider
+
+To use the library you need to put a `ThemeProvider` which come from `@emotion/react` library :
```

Comment:
```suggestion
To use the library you need to put a `ThemeProvider` which come from `@emotion/react` library:
```

It is simple but there is no space before and after ponctuation in english in contrary of french 😉